website/docs/workloads/sysbench-oltp/sysbench-oltp-metrics.md

```diff
@@ -0,0 +1,30 @@
+# Sysbench Workload Metrics
+
+The following document illustrates the type of results that are emitted by the Sysbench workload and captured by the
+Virtual Client for net impact analysis.
+
+### Workload-Specific Metrics
+
+The following metrics are captured during the operations of the Sysbench OLTP workload.
+
+#### Metrics
+
+| Execution Profile     | Test Name | Metric Name | Example Value | Unit |
+|-----------------------|-----------|-------------|---------------|------|
+| PERF-MYSQL-SYSBENCH-OLTP.json | Sysbench OLTP | # read queries | 5503894 |  |
```

Comment:
Just to make sure to randomly scrabble the results a little bit. If you did those on Azure VMs.